Tests/LibWeb/Text/input/Streams/WritableStream-write.html

```diff
@@ -0,0 +1,25 @@
+<script src="../include.js"></script>
+<script>
+    test(() => {
+        const stream = new WritableStream({
+            write(chunk) {
+                return new Promise(resolve => {
+                    println(chunk);
+                    resolve();
+                });
+            },
+        });
+
+        function sendMessage(message) {
+            const writer = stream.getWriter();
+
+            for (const chunk of message) {
+                writer.ready.then(() => writer.write(chunk));
+            }
+
+            writer.ready.then(() => writer.close());
+        }
+
+        sendMessage("Well-hello-friends!");
```

Comment:
shouldn't we wait until all promises in the test are resolved before ending a test?

Reply:
Woops, yes we should, thanks - fixed.